includes/Modules/Analytics.php

```diff
@@ -1196,15 +1196,19 @@ private function print_tracking_opt_out() {
 		if ( ! $this->is_tracking_disabled() ) {
 			return;
 		}
+		$settings    = $this->get_settings()->get();
+		$account_id  = $settings['accountID'];
+		$property_id = $settings['propertyID'];
 
 		?>
 		<!-- <?php esc_html_e( 'Google Analytics user opt-out added via Site Kit by Google', 'google-site-kit' ); ?> -->
 		<?php if ( $this->context->is_amp() ) : ?>
 			<meta name="ga-opt-out" content="" id="__gaOptOutExtension">
 		<?php else : ?>
-			<script type="text/javascript">window["_gaUserPrefs"] = { ioo : function() { return true; } }</script>
+			<script type="text/javascript">window["ga-disable-<?php echo esc_attr( $property_id ); ?>"] = true; </script>
 		<?php endif; ?>
 		<?php
+		do_action( 'googlesitekit_analytics_tracking_opt_out', $property_id, $account_id );
```

Comment:
This action should go into the web-specific `else` above. This was mentioned in the IB but this specific detail was lost. Otherwise we risk emitting invalid AMP markup in that case. This action is web-specific though as there is no way to do this property-specific opt-out in AMP.